### PR TITLE
MAR-1469, MAR-1324 Multicandidate flow

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

**Comment by malmas18** [comment]
the csv generation and also show an edit button when the brief is specialists and the supplier hasn't submitted 3 briefs yet

File: app/buyers/views/buyers.py [hunk 1]
```diff
@@ -424,19 +424,23 @@ def row(r):
...
-                                                                                _external=True
-                                                                                )
-                                if i < len(r.get('attachedDocumentURL', [])) else ''})
+            attached_documents_count = len(r.get('attachedDocumentURL', []))
+            links = ''
+            for i in range(0, attached_documents_count):
+                links = ('{}\n{}'.format(links, url_for('.download_brief_response_attachment',
+                                         framework_slug=brief['frameworkSlug'],
+                                         lot_slug=brief['lotSlug'],
+                                         brief_id=brief['id'],
```
Comment (tobiasbrown) [inline app/buyers/views/buyers.py]: flake8 isn't happy with the formatting of these lines.

File: tests/app/views/test_marketplace.py [hunk 2]
```diff
@@ -465,14 +465,35 @@ def test_existing_seller_submitted_app_dp(self):
...
+        document = html.fromstring(res.get_data(as_text=True))
+        assert_equal(
+            document.xpath('//a')[10].text, 'View your application'
+        )
+
+    def test_has_brief_responses_specialists_view(self):
+        self.login_as_supplier()
+
+        self._data_api_client.find_brief_responses.return_value = {
+            'briefResponses': ['firstResponse', 'secondResponse', 'thirdResponse']
```
Comment (tobiasbrown) [inline tests/app/views/test_marketplace.py]: flake8 isn't happy with this line.

File: app/buyers/views/buyers.py (hunk 1 above)
Comment (malmas18) [reply app/buyers/views/buyers.py]: default line length in flake8 is configured to be 79 characters which differs from the configuration of pep8 (120)

if we r u to use flake8 as the default linter, we should take all the steps necessary for this to happen (builds, local, make, etc)

File: tests/app/views/test_marketplace.py (hunk 2 above)
Comment (malmas18) [reply tests/app/views/test_marketplace.py]: same as above

**Pull request merged**